testing/api/test_api.py

```diff
@@ -1149,10 +1376,40 @@ def test_create_invalid_chars(empty_devices_dir, testrun): # pylint: disable=W06
   print(r.text)
   print(r.status_code)
 
+def test_get_test_modules(testrun): # pylint: disable=W0613
+  """Test the /system/modules endpoint to get the test modules"""
+
+  # Send a GET request to the API endpoint
+  r = requests.get(f"{API}/system/modules", timeout=5)
+
+  # Check if status code is 200 (OK)
+  assert r.status_code == 200
+
+  # Parse the JSON response
+  response = r.json()
+
+  # Check if the response is a list
+  assert isinstance(response, list)
+
+  # Define the expected modules
+  expected_modules = [
```

Comment:
We do not want to hardcode the expected modules. Just verify that a list is returned.